lib/puppet/functions/user_password.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Puppet::Functions.create_function(:user_password) do
+  dispatch :user_password do
+    param 'Variant[Sensitive[String], String]', :password
+    return_type 'Variant[Sensitive[String], String]'
+  end
+
+  def user_password(password)
+    Puppet.debug("Received password: #{password}")
```

Comment:
Can you remove this?

Reply:
yes acceptance tests was failing for puppet 8 so I had added this to debug, sure Ill remove this

Reply:
Removed